bin/pycheck

```diff
@@ -17,25 +17,18 @@ cd "$(dirname "$0")/.."
 
 . misc/shlib/shlib.bash
 
-mypy_folders=(ci misc/python)
+PYRIGHT_VERSION="1.1.323"
```

Comment:
Our Bash style is to use lowercase variable names except for environment variables.

I also think you'll want to pull this version out into a file (perhaps ci/builder/pyright-version) so that you can refer to the version in the Dockerfile, as the CI Docker builder should come with the right version of `pyright` already installed.